shopify/slack-notification-template/web/routes/index.tsx

```diff
@@ -151,17 +165,20 @@ const SlackChannelSelectionForm = ({
                     />
                   }
                 >
-                  {options.length > 0 && (
+                  {options.length > 0 ? (
                     <Listbox
                       onSelect={(value) => {
                         setInputValue(
```

Comment:
is undefined valid? We can always check here and not set the input if it is not (so we don't have to cast as a string below)

Reply:
I'm not able to find an alternative